analyzer/consensus/consensus.go

```diff
@@ -500,20 +499,6 @@ func (m *Main) queueRuntimeRegistrations(batch *storage.QueryBatch, data *storag
 	return nil
 }
 
-func (m *Main) queueRuntimeStatusUpdates(batch *storage.QueryBatch, data *storage.RegistryData) error {
-	runtimeSuspensionQuery := m.qf.ConsensusRuntimeSuspensionQuery()
-	runtimeUnsuspensionQuery := m.qf.ConsensusRuntimeUnsuspensionQuery()
-
-	for _, runtime := range data.RuntimeSuspensions {
-		batch.Queue(runtimeSuspensionQuery, runtime)
```

Comment:
should we remove the `runtime[Un]SuspensionQuery` as well? Or is it used when processing genesis?

Reply:
Good catch, I missed those! Removed now.